packages/data-stores/src/onboard/utils.ts

```diff
@@ -25,3 +25,14 @@ export const goalsToIntent = ( goals: SiteGoal[] ): SiteIntent => {
 
 	return SiteIntent.Build;
 };
+
+export const serializeGoals = ( goals: SiteGoal[] ): string => {
+	// Serialize goals by first + alphabetical order.
+	const firstGoal = goals.find( ( goal ) =>
+		[ SiteGoal.Write, SiteGoal.Sell, SiteGoal.Promote, SiteGoal.DIFM ].includes( goal )
```

Comment:
Is it intentional to exclude `import` and `other` goals from being a valid first goal? Mind that with the current implementation those goals will still be included in the string, and still may happen to be at the beggining of the string if they are the only goals selected.

Reply:
> Is it intentional to exclude import and other goals from being a valid first goal?

I think this is intentional - based on Yan's [explanation here](pdDOJh-qI-p2#serializing-goals-using-first-alphabetical-order).

Reply:
@ivan-ottinger oh I see, thanks for that 👍